__tests__/reducers/resourceValidationHelpers.test.js

```diff
@@ -0,0 +1,480 @@
+// import { updateValueErrors } from "reducers/resources"
+import Config from "Config"
+// import configureMockStore from "redux-mock-store"
+// import thunk from "redux-thunk"
+import { createState } from "stateUtils"
+import { createReducer } from "reducers/index"
+import {
+  // addProperty,
+  // addSubject,
+  addValue,
+  // clearResource,
+  // hideNavProperty,
+  // hideNavSubject,
+  // hideProperty,
+  // removeSubject,
+  // removeValue,
+  // saveResourceFinished,
+  // setBaseURL,
+  // setCurrentEditResource,
+  // setCurrentPreviewResource,
+  // setUnusedRDF,
+  // showNavProperty,
+  // showNavSubject,
+  // showProperty,
+  // loadResourceFinished,
+  // setResourceGroup,
+  // setValueOrder,
+  // clearResourceFromEditor,
+  // saveResourceFinishedEditor,
+  // updateValue,
+  // setCurrentDiffResources,
+  // setVersions,
+  // clearVersions,
+} from "reducers/resources"
```

Comment:
I wasn't sure what I needed to write tests;   at this point, I wonder if I need to have tests for updateValue or any other reducers.

Reply:
I would add a single test that shows that validation is being performed for updateValue (if there isn't one already). But no need to repeat all of the test.